src/dali/plugin/pair_converter/ccxt_kraken_locked.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Neal Chambers
```

Comment:
Just a nit: why not call this just ccxt_kraken.py: is there ever going to be a ccxt_kraken.py that is not locked?